addons/google_calendar/models/res_users.py

```diff
@@ -186,3 +186,17 @@ def check_synchronization_status(self):
                 sync_status = 'sync_stopped'
         res['google_calendar'] = sync_status
         return res
+
+    def _has_any_active_synchronization(self):
+        """
+        Check if synchronization is active for Google Calendar.
+        This function retrieves the synchronization status from the user's environment
+        and checks if the Google Calendar synchronization is active.
+
+        :return: Action to delete the event
+        """
+        sync_status = self.env.user.check_synchronization_status()
+        res = super()._has_any_active_synchronization()
+        if sync_status.get('google_calendar') == 'sync_active':
+            res |= True
+        return res
```

Comment:
Why are you launching check_synchronization_status on self.env.user when you are trying to know if user in self has any active sync ? It feels strange. You will never be able to compute it for another user than the current user. The test is probably not well written if it's working. :thinking: